include/i3_containers.hpp

```diff
@@ -148,6 +158,7 @@ namespace i3_containers
         std::vector<std::uint64_t> focus;                 /**< List of child node IDs in focus order.                 */
         std::vector<node> nodes;                          /**< The tiling child containers of this node.              */
         std::vector<node> floating_nodes;                 /**< The floating child containers of this node.            */
+        fullscreen_mode_type fullscreen_mode;             /**< The fullscreen mode of the container.                  */
```

Comment:
Please move this member above `nodes` to match the order of the `i3` documentation.